### PR TITLE
Added Prometheus configuration to monitor Istio

### PR DESCRIPTION
**Pull request by evanlouie**
- Disabled Istio's prometheus and grafana addons
- Added Prometheus scrape jobs to monitor Istio
- Disabled Istio's Jaeger addon but kept`enableTracing` enabled to still have the mesh generate spans

### CODE REVIEW

**Comment by evanlouie** [comment]
resolves https://github.com/CatalystCode/kubemalt/issues/29

File: config/common.yaml [hunk 1]
```diff
@@ -4,3 +4,127 @@ subcomponents:
     config:
       extraArgs:
         prometheus-url: http://prometheus-server.prometheus.svc.cluster.local
+
+  istio:
+    config:
+      namespace: istio-system # this is created by the Istio component
+      tracing:
+        enabled: false # Provided by other component
```
Comment (timfpark) [inline config/common.yaml]: Consider explaining which other component provides this (fabrikate-jaeger) so its clearer to folks newer to all of this than us.

File: config/common.yaml [hunk 2]
```diff
@@ -4,3 +4,127 @@ subcomponents:
...
+      global:
+        enableTracing: true # Enable global.enableTracing to make use of our manual Jaeger installation
+      prometheus:
+        enabled: false # Provided by other component; Inject the istio monitoring jobs via extraScrapeConfigs in Prometheus chart
```
Comment (timfpark) [inline config/common.yaml]: Consider explaining which other component provides this (fabrikate-prometheus-grafana)

File: config/common.yaml [hunk 3]
```diff
@@ -4,3 +4,127 @@ subcomponents:
...
+      grafana:
+        enabled: false # Provided by other component.
```
Comment (timfpark) [inline config/common.yaml]: Consider explaining which other component provides this (fabrikate-prometheus-grafana)

File: config/common.yaml (hunk 2 above)
Comment (evanlouie) [reply config/common.yaml]: done!

File: config/common.yaml (hunk 3 above)
Comment (evanlouie) [reply config/common.yaml]: done!

**Pull request merged**